### ISSUE TITLE
Problem with channels when using MNIST dataset.

### ISSUE THREAD

**Issue by JoseLlorensRipolles** [quark0/darts#76]
I am trying to do the search on the MNIST dataset, however, I have found some issues related to the images only having 1 channel.

On _cnn/model_search.py_ line 74 :

`      nn.Conv2d(3, C_curr, 3, padding=1, bias=False),
`

Shouldn't that _"3"_ be _init_channels_? If not, could you please give me some briefing on what is init_channels used for?

**Comment by chakkritte** [comment]
@JoseLlorensRipolles You can see the example from PyTorch

https://github.com/pytorch/examples/blob/master/mnist/main.py#L13

`nn.Conv2d(1, C_curr, 3, padding=1, bias=False),`

I think it should be corrected to be 1
